addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -397,13 +397,17 @@ export const editorCommands = {
         const changedElements = [];
         const defaultDirection = editor.options.direction;
         const shouldApplyStyle = !isSelectionFormat(editor.editable, 'switchDirection');
-        for (const block of new Set(selectedTextNodes.map(textNode => closestBlock(textNode)))) {
+        for (const block of new Set(selectedTextNodes.map(textNode => closestBlock(textNode).parentNode.nodeName === "LI" ? closestBlock(textNode).parentNode : closestBlock(textNode)))) {
```

Comment:
Yes let's discuss in 3D! :smile: But it's a special case for lists, so when the text is in a list, we get the list element instead of the closest block